devicons.py

```diff
@@ -147,6 +147,16 @@
     'Images'                           : '',
     'Publique'                         : '',
     'Vidéos'                           : '',
+# Portuguese
+    'Área de trabalho'                 : '',
+    'Documentos'                       : '',
+    'Downloads'                        : '',
```

Comment:
This line doesn't differ from the english definition and thus shouldn't be redefined.